app/serializers/order_serializer.rb

```diff
@@ -61,10 +64,34 @@ def ordered_by_current_user
   def from_today
     object.from_today?
   end
+  
+  def who_ordered
+    object.user.name
+  end
+
+  def current_user_dish
+    if current_user_ordered_dish?
+      object.dishes.where('dishes.user_id = ?', current_user.id).first.name
```

Comment:
You could use `find_by` here (e.g., `object.dishes.find_by(user: current_user).name`)

Reply:
And also, the guard clause would be a pretty good idea:

```
def 
  return "You didn't order any dish." unless sth
  object.dishes.find_by(user: current_user).name.to_s
end
```